packages/ses/src/compartment.js

```diff
@@ -326,6 +332,18 @@ export const makeCompartmentConstructor = (
       sloppyGlobalsMode: false,
     });
 
+    let evaluator;
+
+    if (evalTaming === 'unsafe-no-direct') {
+      evaluator = () => {
+        throw TypeError(
```

Comment:
Does this take care of all three evaluators: `eval`, `Function`, `Compartment` (or just Compartment `evaluate`)?

`'no-eval'` should have the same effect on Compartments as `'unsafe-no-direct'`. Why is it a separate code path? Are there any differences observable from within a Compartment?